service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -291,6 +296,15 @@ public C2DocumentBundle getLastC2DocumentBundle() {
             .orElse(null);
     }
 
+    @JsonIgnore
+    public C2DocumentBundle getMostRecentC2DocumentBundle() {
```

Comment:
minor: It would be better to sort here, then from this method user knows that you want one that is newest. If it is just get(0) it is hard to know what does it mean, is it oldest, newest? But can stay as i tis

Reply:
discussed and leaving as is